src/context/Theme.tsx

```diff
@@ -70,9 +72,15 @@ export const sample_grey = {
 };
 
 // Status Colors
-export type SemanticColor = '#40B763' | '#F5D75E' | '#CA3B55' | '#4FD6FF';
+export type SemanticColor =
+    | '#40B763'
+    | '#2A7942'
+    | '#F5D75E'
+    | '#CA3B55'
+    | '#4FD6FF';
 
 export const successMain: SemanticColor = '#40B763';
+export const successDark: SemanticColor = '#2A7942';
```

Comment:
The _main_ semantic color used to indicate success does not meet the AA level contrast. Even when adjusted in accord with the contrast threshold (i.e., the calculated `theme.palette.success.dark`), the contrast was still unsatisfactory. Therefore, the dark variant of the "success" semantic color was explicitly defined, using the _main_ variant as a base.